script/build

```
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+platforms="darwin/amd64 darwin/arm64 linux/386 linux/amd64 linux/arm linux/arm64 windows/386 windows/amd64"
+
+# clear bin directory
+rm -rf ./bin/*
+
+i=0
+for platform in ${platforms}; do
+    ((i++))
+
+    split=(${platform//// }) # split platform by sep "/"
+    goos="${split[0]}"
+    goarch="${split[1]}"
+    output="benchttp_${goos}_${goarch}" # e.g. benchttp_darwin_amd64
+
+    # add .exe to windows binaries
+    [[ "$goos" == "windows" ]] && output="$output.exe"
```

(I didn't know this syntax at all in Bash!)

---

Could be a bad practice, I'm rather terrible at bash 😬

---

Works because `&&` requires success of previous command. We could also do:

```suggestion
    if [ "$goos" == "windows" ]; then output="$output.exe"; fi
```

(I think so at least 🤔)

---

Is it any better than the short-circuit?

---

No idea 🤷‍♂️ I have no preference.

---

Let's save an additional commit for the planet! 🚀 🌍